29787 text wrap (#30435)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

To ensure compatibility with these changes, the [Vendor-prefixed transform properties](#vendor-prefixed-transform-properties) and the [Vendor-prefixed transition properties](#vendor-prefixed-transition-properties) are disabled in the Nightly release.
These changes are described in the following sections.

### text-wrap: balance & stable values

> The [`text-wrap`](/en-US/docs/Web/CSS/text-wrap) CSS property values `balance` and `stable` allow the layout of content to be wrapped for short content in a balanced manner and for editable content without rewrapping while the user is editing.

Reviewer: Subjective improvements...still feels a bit awkward to me...might be worth breaking the short descriptions of these property values into two sentences. wdyt?

```suggestion
The [`text-wrap`](/en-US/docs/Web/CSS/text-wrap) CSS property values `balance` and `stable` allow the layout of short content to be wrapped in a balanced manner and for editable content to not reflow while the user is editing it, respectively.
```
